casper/src/main/formal_verification/tla+/RChainCasper.md

```diff
@@ -0,0 +1,10 @@
+# RChain's Casper Implementation
+
+Features of RChain's Casper implementation:
+- Latest estimate driven estimator: multi-parent GHOST
+- GHOST fork-choice rule (https://github.com/ethereum/wiki/wiki/White-Paper#modified-ghost-implementation)
+- validator rotation
+- slashing/equivocation
+- post-unbonding stake holding time (prevents long range attack)
+- safe from army of ants attack/byzantine generals problem/whales (centralization)
+- block production rewards/transaction fees
```

Comment:
I have never played with tla before. Could you document how to play with the tla or some related links?

Reply:
Yes. I'll include a README explaining how to get set up and use the tools.

Reply:
Is it good style to document this stuff in a README? What is the preferred method of documentation?

Reply:
https://github.com/rchain/rchain/blob/dev/README.md or https://github.com/rchain/rchain/blob/dev/DEVELOPER.md?
I think you can put them in RChainCasper.md.
Just make sure link one of the docs above to your document

Reply:
I am not sure if we have documentation style.